hack/pseudo-version/pseudo-version.go

```diff
@@ -44,19 +45,21 @@ func main() {
 		log.With(zap.Error(err)).Fatalf("Failed to get parsed branch name")
 	}
 
-	rawBranch, err := gitc.BranchName()
-	if err != nil {
-		log.With(zap.Error(err)).Fatalf("Failed to get branch name")
-	}
-
 	if *base == "" {
-		_, versionTag, err := gitc.FirstParentWithVersionTag()
+		rootPath, err := gitc.Path()
 		if err != nil {
-			log.With(zap.Error(err)).Warnf("Failed to find base version. Using default.")
-			versionTag = ""
+			log.With(zap.Error(err)).Fatalf("Failed to get git root path. Using default base version.")
+			//
+		} else {
+			versionTag, err := os.ReadFile(path.Join(rootPath, "/version.txt"))
```

Comment:
nit for 🪟

```suggestion
			versionTag, err := os.ReadFile(filepath.Join(rootPath, "version.txt"))
```

Reply:
[Windows does not care whether you use `/` or `\`](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#canonicalize-separators) ;)
`C:/Windows\\system32/cmd.exe` for example is perfectly acceptable as path and works fine.

Though yes, `filepath` is better.